eng/versioning/version_client.txt

```diff
@@ -43,7 +43,6 @@ com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 unreleased_com.azure:azure-core;1.4.0-beta.1
-unreleased_com.azure:azure-core-amqp;1.1.0-beta.1
```

Comment:
unreleased_com.azure:azure-core-amqp should not be removed until com.azure:azure-core-amqp has been released and the dependency updated.